packages/terra-functional-testing/src/commands/validates/element.js

```diff
@@ -0,0 +1,27 @@
+const accessibility = require('./accessibility');
+
+/**
+ * An assertion method to assert the page is accessible and the screenshot comparison result is within
+ * the mismatch tolerance.
+ *
+ * This should be used within a Mocha `it` block.
+ *
+ * @param {Object} [options] - the test options
+ * @param {string} [options.testName] - the test case name.
+ * @param {Object} [options.rules] - the axe rules to use to assert accessibility.
+ * @param {Number} [options.misMatchTolerance] - the mismatch tolerance for the screenshot comparison.
+ * @param {string} [options.selector] - the element selector to use for the screenshot comparison.
+ */
+const element = (options = {}) => {
+  const { rules } = options;
+
+  accessibility({ rules });
```

Comment:
this isn't necessary, just personal preference
```suggestion
  accessibility({ options?.rules });
```

Reply:
One thing to watch out with optional chaining, it must be transpiled for node versions less that 14.0.0